src/locales/zh/messages.po

```diff
@@ -353,7 +353,7 @@ msgstr "折扣率"
 
 #: src/components/Create/index.tsx
 msgid "Discount rate disabled while funding cycle duration is 0."
-msgstr "筹款周期时长设置为 0 时，折扣率选项将会禁用。"
+msgstr "筹款周期持续时间设置为 0 时，折扣率选项将会禁用。"
```

Comment:
**issue**: we shouldn't include translation file updates in PRs, right?

side note, we should add a CI check for this

Reply:
Yeah so with this, I pulled the Crowdin branch into this one so I could more easily find the texts that hadn't been marked for translation, that's why these diffs are here. @peripheralist, I've pulled in the latest from Crowdin to the Crowdin branch, and then from Crowdin branch to this branch - so just merge this one I think, no need to worry about the Crowdin branch.